packages/backend/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "esnext",
     "module": "esnext",
-    "moduleResolution": "Node",
+    "moduleResolution": "bundler",
```

Comment:
this seems odd ? why is this required ?

Reply:
I confirm that building the extension, and installing from a registry works well, so this does not seems to affect the build output.